zntrack/core/dvcgraph.py

```diff
@@ -47,26 +47,42 @@ def handle_deps(value) -> list:
     return script
 
 
-def get_dvc_arguments(options: dict) -> list:
-    """Get the activated options
+@dataclasses.dataclass
+class DVCRunOptions:
+    """Collection of DVC run options
 
-    Returns
-    -------
-    list: A list of strings for the subprocess call, e.g.:
-        ["--no-commit", "--external"]
+    References
+    ----------
+    https://dvc.org/doc/command-reference/run#options
     """
-    out = []
 
-    for dvc_option, value in options.items():
-        if value:
-            out.append(f"--{dvc_option.replace('_', '-')}")
-        else:
-            if dvc_option == "no_exec":
-                log.warning(
-                    "You will not be able to see the stdout/stderr "
-                    "of the process in real time!"
-                )
-    return out
+    no_commit: bool
```

Comment:
Because these are public attributes they could be documented somewhere.

Reply:
I reference https://dvc.org/doc/command-reference/run#options here because I don't think copy&paste from the official DVC documentations would help here. The official docs describe the attributes better than all I could offer here. But I will add a comment under Attributes